eng/pipelines/templates/steps/build.yml

```diff
@@ -48,12 +48,29 @@ steps:
   - template: /eng/pipelines/templates/steps/set-artifact-packages.yml
     parameters:
       Artifacts: ${{ parameters.Artifacts }}
+  - ${{if and(in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'Manual'), eq(variables['System.TeamProject'], 'internal'))}}:
+    - task: AzureCLI@2
+      inputs:
+        azureSubscription: azure-sdk-tests
```

Comment:
@weshaggard in my old PR we decided to use a container from the EngSys subscription. However, I could only find service connection for the Tests subscription.  I created a container in the Tests subscription to test the PR and it worked https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4065915&view=logs&j=592d6fa2-9024-5c63-e3c6-2497f0c9c2f5&t=e138b8cb-fa3a-5e7f-1765-bba06204455e&l=679

Is it possible to add a service connection to EngSys so that we can use the original planned container? Or use this one from Tests sub (need to ensure it is not deleted by cleanup automation though)

Reply:
/cc @benbp